### ISSUE TITLE
FritzWlan doesn't work 

### ISSUE THREAD

**Issue by thost96** [andig/pimatic-fritz#36]
Hi, after upgrading pimatic-fritz to version 0.5.4 today, the FritzWlan Class device doesn't display the switch and the wifi information anymore. Only the device name is shown corretly. 

with version 0.5.3 it works fine.

**Comment by andig** [comment]
Then it must be the difference between fritzapi 0.8 and 0.9.2. Could you check with a small test program which version introduced the breaking change for you?

**Comment by andig** [comment]
Also: are you accessing your fb over https? If yes- using a "real" certificate?

Could you check if changing line 18 like this fixes the problem for you:

```
var request = require('request').defaults({ strictSSL: false }); // be less strict about SSL errors

**Comment by ikewestbrook** [comment]
same here! where do these changes need to be applied?
by the way, i get error message: error [pimatic]: unexpected newline

**Comment by andig** [comment]
> same here! where do these changes need to be applied?

This would need to go to `node_modules/fritzapi/index.js`

> by the way, i get error message: error [pimatic]: unexpected newline

I would need more details- and a new issue please

**Comment by leader21** [comment]
i wasn't able to switch my dect200 plug. there has been an error with the indention. the pluginwas not able to compile after the update. with the simple fix it should wok again.
check my pull request https://github.com/andig/pimatic-fritz/pull/38
and my issue andig/pimatic-fritz#37

**Comment by andig** [comment]
Released 0.5.5. If this is not working please test the suggestion above for ssl security and let me know.

**Comment by thost96** [comment]
Sorry for my delayed feedback. I'm not using https and with pimatic-fritz version 0.5.5 it's working again.
